### DISCUSSION TITLE
v18.1.0

### DISCUSSION THREAD

**Discussion by Kyon147** [Kyon147/laravel-shopify#161] [Announcements]
## What's Changed
* Update the namespace comment by @Kyon147 in https://github.com/Kyon147/laravel-shopify/pull/128
* bugfix: Add missing foreign key for shop/charge relationship by @ClaraLeigh in https://github.com/Kyon147/laravel-shopify/pull/140
* Feature: Manually map a webhook to a dispatchable class by @ClaraLeigh in https://github.com/Kyon147/laravel-shopify/pull/127
* Bump default shopify API version by @npabisz in https://github.com/Kyon147/laravel-shopify/pull/132
* Extend session token class api by @stidges in https://github.com/Kyon147/laravel-shopify/pull/141
* Implement leeway for nbf and iat in session token by @stidges in https://github.com/Kyon147/laravel-shopify/pull/147

## New Contributors
* @ClaraLeigh made **another** contribution in https://github.com/Kyon147/laravel-shopify/pull/140
* @npabisz made their first contribution in https://github.com/Kyon147/laravel-shopify/pull/132
* @stidges made their first contribution in https://github.com/Kyon147/laravel-shopify/pull/141

**Full Changelog**: https://github.com/Kyon147/laravel-shopify/compare/v18.0.1...v18.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Kyon147/laravel-shopify/releases/tag/v18.1.0'>v18.1.0</a>.</em>

**Comment by ClaraLeigh** [comment]
Typo: not my first contrib! :P

**Reply by Kyon147** [reply]
@ClaraLeigh I did not think it was, but it came up when using the "auto notes" ha
